charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ All changes to this chart will be documented in this file.
 * Enable the setup of ReadOnlyRootFilesystem in the security contexts
 * Support basic chart installation on Openshift
 * Support Openshift Route
+* Support Openshift SCCv2 by default when Openshift.enabled=true
+* Deprecate Openshift.createSCC
```

Comment:
Given that `Openshift.createSCC` gets deprecated, why don't we set the default value to false?

Reply:
i would prefer not to change the default to keep the compatibility and not change the behavior for the few users that might be relying on this feature.